### DISCUSSION TITLE
Airflow server postgres DB not starting and throwing errors

### DISCUSSION THREAD

**Discussion by keerthivasan5678** [apache/airflow#36104] [Q&A]
Hi all ,

I am getting the below error in airflow version 2.7.3 when i tried to initiate the db using airflow db init command

we are using postgres DB with local executor.

The server was working fine and suddenly this happened.

 conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" (127.0.0.1), port 5432 failed: Connection refused
        Is the server running on that host and accepting TCP/IP connections?

Can anyone share your thoughts about this

**Comment by potiuk** [comment]
Sure. Exactly what in the message your DB is down. you need to manage your DB, know the resources it has, check the logs and all the stuff when you manage DB.

Generally speaking if you chose to use postgres run "internally" - you are 100% responsible for all those things and you need to become a DB admin and understand all the aspects of it as your deployment manager  - but recommended setup for anything close to production is to use some managed version of Posrgres.

**Reply by keerthivasan5678** [reply]
Thanks Jarek you were right the DB was down and once i brought db back to online the issue got fixed.
